static/app/views/settings/projectSourceMaps/sourceMapsList.tsx

```diff
@@ -69,7 +70,19 @@ function mergeReleaseAndDebugIdBundles(
   return [...debugIdUploads, ...releaseUploads] as SourceMapUpload[];
 }
 
-function useSourceMapUploads({organization, project, query, cursor}) {
+interface UseSourceMapUploadsProps {
+  cursor: string | undefined;
```

Comment:
shouldn't this be optional?

Reply:
at the moment they're always provided. can be made optional if someone needs that later